README.md

```diff
@@ -0,0 +1,80 @@
+* **`applies`**: determines which hosts the alert applies to; putting `true` will end up creating as many alerts as the number of distinct `name`s your alert expands into. Interferon will iterate through all the hosts generated by `host_sources` and test them agains the expression here.
+* **`notify.people`**: an array of the people who will be receiving the alert.
+* **`notify.groups`**: an array of groups who will be receiving the alert. The group mapping will be based on the output of `group_sources`. The `filesystem` group_source module will read the groups in the `groups` directory.
+* **`metric`**: The Datadog metric alert query.
+* **`silenced`**: The Datadog silenced setting.
+* **`timeout`**: The Datadog timeout setting in seconds (gets converted to `timeout_h`).
+
+### destinations
+This directory contains additional modules for `destinations`.
+
+### groups
+We only have a single group_source called `filesystem`, which reads groups in YAML files from the filesystem (specifically this `groups`). Each file contains the following keys:
+* **`name`**: Name of the group (used in alerts).
+* **`people`**: List of people in the group. 
+* **`alias_for`**: Name of another group that this group is an alias for. Useful for transistion periods when renaming groups.
+
+To post to a slack channel, make sure that the slack integration is configured to include that channel: https://app.datadoghq.com/account/settings#integrations/slack
+
+### group_sources
+This directory contains additional modules for `group_sources`. Each group_source should be a class which implements the following methods.
+* **`intialize(options)`**: Called for initializing the group_source and reading the options passed in from `config.yml`.
+* **`list_groups`**: Called to get the group mapping generated by this module. Should return a hash contain group name to people.
+
+### host_sources
+This directory contains additional modules for `host_sources`. Each host_source should be a class which implements the following methods:
+* **`intialize(options)`**: Called for initializing the group_source and reading the options passed in from `config.yml`.
+* **`list_hosts`**: Called to get the host list generated by this module. Should return a list of hashes that can be referenced using `@hostinfo` in alerts.
+
+### Tests
+Tests can be run via run via `rake spec` (to test additional code/modules) and `rake test` (to test the alert files). There is a datadog syntax checker built into `rake test` that will check for valid Datadog syntax.
+
+There are additional tests in `script/pre-commit` and `test/check_syntax.sh` that can be run to check the syntax of other files.
+
+### Deploying
+When committing to the alerts repository, you should ensure that `rake test` is run to check the syntax of the datadog metrics.
+
+Once built and deployed, interferon can be run via the following commands assuming the `config.yml` file is placed in `/etc/alerts`.
```

Comment:
i wonder if we should tell people to put files into `/etc` -- an example like `bundle exec interferon -c ./config.yml` would work just as well.